prime-dotnet-webapi/Controllers/JobsController.cs

```diff
@@ -77,5 +80,20 @@ public async Task<ActionResult> PopulateTransactionLogTemp(int numberOfDays)
             var result = await _reportingService.PopulateTransactionLogTempAsync(numberOfDays);
             return Ok(result);
         }
+
+        // POST: api/jobs/populate/organization-registration-id
+        /// <summary>
+        /// execute job to update organization registration ID where the registration ID is missing
```

Comment:
Indicate that number of organizations updated is returned

Reply:
Update as suggested.